src/main/cpp/vision/DragonQuest.cpp

```diff
@@ -13,77 +13,86 @@
 // OR OTHER DEALINGS IN THE SOFTWARE.
 //====================================================================================================================================================
 #include "units/time.h"
-#include "vision/DragonQuest.h"
 #include "utils/AngleUtils.h"
+#include "vision/DragonQuest.h"
+#include "utils/DragonField.h"
 
 DragonQuest *DragonQuest::m_dragonquest = nullptr;
```

Comment:
If this isn't a singleton anymore, we should remove the static pointer and the static GetInstance method